scripts/payouts/multi-token/create_full_recipients_setup.py

```diff
@@ -0,0 +1,169 @@
+            deploy_config.trusted_caller,
+            deploy_config.limit,
+            deploy_config.period,
+            deploy_config.tokens,
+            deploy_config.recipients,
+            deploy_config.titles,
+            deploy_config.spent_amount,
+            tx_params,
+        )
+        print(tx.events)
+
+        allowed_recipients_registry_address = tx.events["AllowedRecipientsRegistryDeployed"]["allowedRecipientsRegistry"]
+        allowed_tokens_registry_address = tx.events["AllowedTokensRegistryDeployed"]["allowedTokensRegistry"]
+        top_up_allowed_recipients_address = tx.events["TopUpAllowedRecipientsDeployed"]["topUpAllowedRecipients"]
+        add_allowed_recipient_address = tx.events["AddAllowedRecipientDeployed"]["addAllowedRecipient"]
+        remove_allowed_recipient_address = tx.events["RemoveAllowedRecipientDeployed"]["removeAllowedRecipient"]
+
+    else:
+        allowed_tokens_registry_address = deploy_config.tokens_registry
+
+        tx = allowed_recipients_builder.deployAllowedRecipientsRegistry(
+            deploy_config.limit,
+            deploy_config.period,
+            deploy_config.recipients,
+            deploy_config.titles,
+            deploy_config.spent_amount,
+            deploy_config.grant_rights,
+            tx_params,
+        )
+
+        allowed_recipients_registry_address = tx.events["AllowedRecipientsRegistryDeployed"]["allowedRecipientsRegistry"]
+
+        tx = allowed_recipients_builder.deployAddAllowedRecipient(
+            deploy_config.trusted_caller,
+            allowed_recipients_registry_address,
+            tx_params,
+        )
+
+        add_allowed_recipient_address = tx.events["AddAllowedRecipientDeployed"]["addAllowedRecipient"]
+
+        tx = allowed_recipients_builder.deployRemoveAllowedRecipient(
+            deploy_config.trusted_caller,
+            allowed_recipients_registry_address,
+            tx_params,
+        )
+
+        remove_allowed_recipient_address=tx.events["RemoveAllowedRecipientDeployed"]["removeAllowedRecipient"]
+
+        tx = allowed_recipients_builder.deployTopUpAllowedRecipients(
+            deploy_config.trusted_caller,
+            allowed_recipients_registry_address,
+            deploy_config.tokens_registry,
+            tx_params,
+        )
+
+        top_up_allowed_recipients_address = tx.events["TopUpAllowedRecipientsDeployed"]["topUpAllowedRecipients"]
+
+    log.ok("Allowed recipients Easy Track contracts have been deployed!")
+    log.nb("Deployed AllowedRecipientsRegistry", allowed_recipients_registry_address)
+    log.nb("Deployed AllowedTokensRegistry" if new_token_registry_is_required else "Used AllowedTokensRegistry", allowed_tokens_registry_address)
+    log.nb("Deployed AddAllowedRecipient", add_allowed_recipient_address)
+    log.nb("Deployed RemoveAllowedRecipient", remove_allowed_recipient_address)
+    log.nb("Deployed TopUpAllowedRecipients", top_up_allowed_recipients_address)
+
+    log.br()
```

Comment:
Maybe it’s worth running the acceptance test right after deployment to avoid filling in the config twice?